docs/specs/clients/notify/data-structures.md

```diff
@@ -26,10 +26,10 @@
   title: string,
   // Long messages used in the body of the notification
   body: string,
-  // Image URL used to display with the notification. If null, the app's icon from Notify Config is used instead
-  icon: string | null,
-  // Redirect URL for call-to-action related to notification
-  url: string | null,
+  // Image URL used to display with the notification. If empty, the app's icon from Notify Config is used instead
+  icon: string,
```

Comment:
To make it clear to expect this to be an empty string. The other variables will be guaranteed by Notify Server not to be empty.

```suggestion
  icon: string | "",
```

Reply:
Isn't this confusing?

Reply:
@devceline thoughts?

Reply:
my thoughts are above

Reply:
My UI didn't update, lol